spec/views/layouts/listnav/_ems_cloud.html.haml_spec.rb

```diff
@@ -10,11 +10,11 @@
   end
 
   it "Flavors link for Openstack cloud manager uses restful path" do
-    record =  ManageIQ::Providers::Openstack::CloudManager.new(:name => "Test Cloud")
+    record = FactoryGirl.create(:ems_openstack, :id => 10)
```

Comment:
You never want to create records with hardcoded ids.  It can cause sporadic failures if 2 records collide.  It also completely ignores regions.  You can get the id *out* of the created record by calling `record.id`